src/cmd_tool.rs

```diff
@@ -6,102 +6,119 @@ use simulation_curator::gtfs::ShapePoint;
 use simulation_curator::cell_data;
 use simulation_curator::nes_simulation;
 use simulation_curator::nes_simulation::create_single_fog_layer_topology_from_cell_data;
+use clap::Parser;
 
-fn main() -> Result<()> {
-    //todo: take command line args
+/// Program to generate topology change events
+#[derive(Parser, Debug)]
+#[command(version, about, long_about = None)]
+struct Args {
+    /// Path to the gtfs database
+    #[arg(short, long, default_value = "gtfs_vbb.db")]
+    dbPath: String,
+
+    ///  The time of the day from when the schedule needs to be selected
+    #[arg(short, long, default_value = "08:00:00")]
+    start_time: String,
+
+    /// The time of the day till when the schedule needs to be selected
+    #[arg(short, long, default_value = "09:00:00")]
+    end_time: String,
+
+    /// The day of the week for which the schedule needs to be selected
+    #[arg(short, long, default_value = "monday")]
+    day: String,
+
+    /// The time interval in seconds that need to be represented by one second. This parameter allows us to speedup the time to increase the rate of topology changes.
+    #[arg(short, long, default_value_t = 20)]
+    batch_interval_size_in_seconds: u64,
+
+    /// The frequency at which the topology changes need to be produced
+    #[arg(short, long, default_value_t = 4)]
+    change_frequency_in_seconds: u64,
+
+    /// Path to the file where fixed_topology.json will be produced
+    #[arg(short, long, default_value = "fixed_topology.json")]    
+    topology_path: String,
+
+    /// Path to the file where topology_updates.json will be produced
+    #[arg(short, long, default_value = "topology_updates.json")]
+    topology_updates_path: String, 
+
+    /// Path to the file where geo.json will be produced
+    #[arg(short, long, default_value = "geo.json")]
+    geo_json_path: String,
+
+    /// Name of the csv file containing OpenCelliD data.
+    #[arg(short, long, default_value = "OpenCelliDGermanyData.csv")]
+    open_cell_id_data_loc: String,
+
+    /// The number of people connected to the base station.
```

Comment:
"The minimum number of measurements required for a cellular base station to be included in the experiment"